docs/vision/strategy-2030.md

```diff
@@ -9,156 +9,139 @@ slug: /vision/strategy-2030
+## Foundation: Governance
 
-| Metric | Target (2030) | Rationale |
-| :--- | :--- | :--- |
-| **L1 Transaction Finality** | P95 finality time `< 5 seconds` | Guarantee rapid, secure settlement essential for financial applications. |
-| **Hydra Head TPS Capacity** | `> 10,000` Transactions Per Second (TPS) | Metric for L2 performance required for competitive Web3 gaming and payments. |
-| **Protocol Stability** | 100% Uptime | Maintain best-in-class operational reliability. |
-| **Research Artifact Velocity** | 10 Papers published/in-review annually | Measure the productivity of IOR's academic network. |
+Cardano governance must be hard to capture, easy to use, and paced. This builds directly on Cardano’s tripartite model of DReps, Constitutional Committee, and SPOs.
 
-### F.2. Interoperability and Security
+### G.1. Incentivized & Accessible Governance
 
-Pioneering cross-chain communication and preparing the protocol for next-generation threats and privacy demands.
+| Focus Area | Strategy | Expected Enhancement |
+| :---- | :---- | :---- |
+| **Role-based incentives for DReps, SPOs & CC** | Fund DReps, SPOs and the Constitutional Committee based on observable activity (votes, rationales, attendance) rather than single binary metrics. | Broader, more diverse participation; less reliance on large-stake actors. |
+| **Governance accessibility tools** | Maintain and develop tools to register as DRep, delegate, vote, submit/view governance actions, and message delegators. | Frictionless participation → higher legitimate turnout. |
 
-| Focus Area | Description | Expected Enhancement |
-| :--- | :--- | :--- |
-| **Interchains** | Establish Cardano as a secure hub for cross-chain transactions by supporting scalable and private multi-chain Decentralized Applications (DApps). | Secure, efficient value transfer between blockchains and legacy systems. |
-| **Core Zero-Knowledge Capabilities** | Standardize modular, updatable Zero-Knowledge Proof (ZKP) infrastructure to enhance privacy and security across all L1 and L2 activity. | Leadership in privacy-preserving solutions and competitive security. |
-| **The Post-Quantum Landscape** | Prepare Cardano with quantum-resistant algorithms for Verifiable Random Functions (VRFs) and threshold signatures. | Proactive defense against risks posed by future quantum computing capabilities. |
+### G.2. Turnout-Aware Voting with Delegator Safeguard
 
-**Key Performance Indicators (KPIs)**
+| Focus Area | Strategy | Expected Enhancement |
+| :---- | :---- | :---- |
+| **Adaptive / turnout-aware thresholds** | Apply turnout-aware approval (like adaptive quorum biasing) so low-turnout proposals need higher yes-ratios. | Prevents “quiet” passes and aligns decisions with actual participation. |
+| **Delegator override of DRep votes** | Allow ADA holders to override their DRep on a per-proposal basis without redelegating. | Protects against captured/inactive DReps while keeping delegation convenient. |
```

Comment:
I agree with this on a layer 2 level, but at the layer 1 it introduces too much complexity into the ledger for very little value.